### PR TITLE
(AP-15) [i18n.ts, packagejson, AccountForm, LoginForm, Newsletter, De…

### PR DESCRIPTION
**Pull request by DenisJankelaic**
…leteAccountModal, useShippingAddressForm, useContactForm, useChangePassword, useEditCustomer]

- Added yup locale. Deleted unnecessary error messages from yup objects.

### CODE REVIEW

File: src/lib/components/Customer/AccountForm/AccountForm.tsx [hunk 1]
```diff
@@ -53,19 +53,16 @@ const AccountForm: FunctionComponent = () => {
...
     const schema = yup.object({
-        firstName: yup.string().required(t(errorMessages.required)),
-        lastName: yup.string().required(t(errorMessages.required)),
-        email: yup
-            .string()
-            .required(t(errorMessages.required))
-            .email(t(errorMessages.email)),
+        firstName: yup.string().required(),
+        lastName: yup.string().required(),
+        email: yup.string().required().email(t(errorMessages.email)),
```
Comment (DenisJankelaic) [inline src/lib/components/Customer/AccountForm/AccountForm.tsx:58]: Leftover.

File: src/lib/drivers/i18n.ts [hunk 2]
```diff
@@ -1,21 +1,34 @@
-import i18n from 'i18next';
+import i18n, { TFunction } from 'i18next';
```
Comment (mraivaras) [inline src/lib/drivers/i18n.ts:1]: import { use, TFunction } from 'i18next';

Comment (DenisJankelaic) [reply src/lib/drivers/i18n.ts:1]: ... tai, kad nėra tokio importo :D

Comment (mraivaras) [reply src/lib/drivers/i18n.ts:1]: wtf, tai kam tada ta TFunction padarė, kaip atskira exporta :facepalm:

Comment (DenisJankelaic) [reply src/lib/drivers/i18n.ts:1]: Typescripto enjoyer'iams, kad type'a turėtų :D

**Review by AndriusUkelis — APPROVED** [review]

**Review by mraivaras — APPROVED** [review]

**Pull request merged**